src/components/pages/blog/blog-pagination.module.scss

```diff
@@ -0,0 +1,62 @@
+ul { 
```

Comment:
Can you please run prettier on this file? It looks like some of the spacing is off